docs/fundamentals/networking/telemetry/metrics.md

```diff
@@ -17,16 +17,20 @@ They are also [multi-dimensional](../../../core/diagnostics/metrics-instrumentat
 > [!TIP]
 > For a comprehensive list of all built-in instruments together with their attributes, see [System.Net metrics](../../../core/diagnostics/built-in-metrics-system-net.md).
 
-## Collect System.Net metrics
+## Collecting System.Net metrics
```

Comment:
We've been trained not to use -ing words in headings and titles - see for example https://review.learn.microsoft.com/help/platform/metadata-required-articles?branch=main#title.